### PR TITLE
BaSP-TG-7:  Employee Entity

### PR DESCRIPTION
**Pull request by CristianAlcober**
This entity has the following:

- [x] A Models.js file created for Employee.

- [x] A Controllers.js file created for Employee.

- [x] A Routes.js file created for Employee.

- [x] A Validations.js file created for Employee.

### CODE REVIEW

**Review by GinaSchP — COMMENTED** [review]
falta la validación antes del create. También borrar los comentarios

**Review by AgustinCarthery — APPROVED** [review]

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
Deberiamos cambiar las responses de la validacion de JOI (ver slack)

File: src/validations/employees.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    status: Joi.boolean().optional(),
+    role: Joi.string().valid('DEV', 'QA', 'PM').required(),
+    project: projectSchema,
+  });
+
+  const validate = employeeValidation.validate(req.body);
+  if (validate.error) {
+    return res.status(400).json({
+      msg: 'There was an error with the validation',
+      error: validate.error.details[0].message,
```
[the remark this answers is not in the document]
Comment (CristianAlcober) [reply src/validations/employees.js]: Just updated!

**Review by GuidoCerioni — APPROVED** [review]
Nice! Well done.

**Review by AgustinCarthery — APPROVED** [review]

**Pull request merged**